File: CPP/land.cpp
#include <iostream>
using namespace std ;

#include "land.h"

void Land::render() {

}

void Land::Interact(int i) {
	if (i == 0){
		//Grow
		this->setGrass(true) ;
	}else if (i == 1) {
		//Be Eaten
		this->setGrass(false) ;
	}
}

bool Land::getGrass() {
	return grass ;
}

void Land::setGrass(bool _grass) {
	grass = _grass ;
}

bool Land::GetAtribute(int i) {
	if (i == 0){
		return grass ;
	}
	return false ;
}

File: main/land/Land.java
package main.land;
//#include "cell.h"
import main.src.Cell;

public class Land extends Cell{

private boolean grass ;
		
public void render() {

}

public void interact(int i) {
	if (i == 0){
		//Grow
		this.setGrass(true) ;
	}else if (i == 1) {
		//Be Eaten
		this.setGrass(false) ;
	}
}

public boolean getGrass() {
	return grass ;
}

public void setGrass(boolean _grass) {
	grass = _grass ;
}

public boolean getAtribute(int i) {
	if (i == 0){
		return grass ;
	}
	return false ;
}


}
